app/Http/Controllers/WikiController.php

```diff
@@ -185,8 +187,16 @@ public function getWikiDetailsForIdForOwner(Request $request): \Illuminate\Http\
       ->with('wikiDbVersion')
       ->with('publicSettings')->first();
 
-        $res['success'] = true;
-        $res['data'] = $wiki;
+        $res = [
+            'success' => true,
+            'data'    => $wiki,
+        ];
+
+        data_set($res, 'data.domain', 
```

Comment:
I'm not sure I've seen this pattern before: what's the advantage of this over just returning it unchanged?

Reply:
you mean the `data_set` function? It's a Laravel helper https://laravel.com/docs/8.x/helpers#method-data-set

I was under the impression that PHP is unhappy if you use unset indices in arrays but I just tested it and it seems that's only true for accessing, it seems to just set the indices if they are not there, which makes sense. I'll change it for simplicity